### PR TITLE
Sanitizing LINE

### PR DESCRIPTION
**Pull request by AdrienFines**
Improve Meetup CSS Selector and removed Meetup JS sanitizer 
Sanitize LINE Privacy Policy

### CODE REVIEW

File: services/Meetup.json [hunk 1]
```diff
@@ -3,17 +3,11 @@
...
-            ]
+            "contentSelector": ".article-info"
         },
         "privacy": {
             "location": "https://help.meetup.com/hc/en-us/articles/360044422391",
-            "contentSelector": ".article",
-            "filters": [
-                "removeFooter"
-            ]
+            "contentSelector": ".article-info"
```
Comment (Ndpnt) [inline services/Meetup.json:10]: This selector removes the document title. The previous one was better.

File: services/Meetup.json [hunk 2]
```diff
@@ -3,17 +3,11 @@
     "documents": {
         "tos": {
             "location": "https://help.meetup.com/hc/en-us/articles/360027447252",
-            "contentSelector": ".article",
-            "filters": [
-                "removeFooter"
```
Comment (Ndpnt) [inline services/Meetup.json:8]: This filter was necessary to remove the link at the end of the document 
`[Follow](/hc/en-us/articles/360027447252-Terms-of-Service/subscription.html "Opens a sign-in dialog")`

File: services/LINE.json [hunk 3]
```diff
@@ -3,11 +3,14 @@
...
         "tos": {
             "location": "https://terms.line.me/line_terms/?lang=en",
-            "contentSelector": ".mdContents01Body"
+            "contentSelector": ".MdTerms01"
         },
         "privacy": {
             "location": "https://terms.line.me/line_rules/?lang=en",
-            "contentSelector": ".mdContents01Body"
+            "contentSelector": ".MdTerms01 > h1:nth-child(1)",
+	    "filters": [
```
Comment (Ndpnt) [inline services/LINE.json:11]: There are space and tabs mixed, please use only spaces of length 2.

File: services/Meetup.json (hunk 1 above)
Comment (AdrienFines) [reply services/Meetup.json]: Ok thanks, thought the title wasn't necesary, I reverted the commit !

File: services/LINE.json (hunk 3 above)
Comment (AdrienFines) [reply services/LINE.json:11]: I will look a it soon, don't understand why on my fork it seems fine

Comment (AdrienFines) [reply services/LINE.json]: I think it is solved now

Comment (Ndpnt) [reply services/LINE.json]: Yep I saw that we have inconsistent tab width. It's ok for now, I will fix it in another PR :).

**Comment by Ndpnt** [comment]
Merged after rebased master

**Pull request closed without merging**